packages/btcindexer/src/btcindexer.ts

```diff
@@ -446,4 +454,31 @@ export class Indexer implements Storage {
 			};
 		});
 	}
+
+	async registerBroadcastedNbtcTx(
```

Comment:
Should we check if the transaction itself is valid?

Reply:
No need for that, we will do it when processing the transactoin in a cron jobs

Reply:
But what if somebody just try to spam us?

Reply:
we will only accept traffic from our frontend so no need